PractiTest/src/main/java/api/practitest/PractiTestAPI.java

```diff
@@ -0,0 +1,61 @@
+package api.practitest;
```

Comment:
Since it's not a generic class like `RequestFactory`, let's move this class to `com.practitest.examples` package.